src/www/tsconfig.json

```diff
@@ -1,9 +1,7 @@
 {
-  "extends": "../tsconfig.json",
+  "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "../../www"
   },
-  "include": [
-    "**/*.ts"
-  ]
+  "exclude": ["../../www/**/*"]
```

Comment:
It's intermittent, but without these excludes you get this weird error in VSCode saying it's trying to overwrite its own input files.

Reply:
(moved to the root tsconfig as just `www`)